google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/Exceptions.java

```diff
@@ -225,7 +225,7 @@ public static class AppendSerializtionError extends RuntimeException {
     private final String streamName;
 
     public AppendSerializtionError(String streamName, Map<Integer, String> rowIndexToErrorMessage) {
```

Comment:
Hi, I just noticed that this class name is also spelled incorrectly.

Reply:
Err... too bad. Maybe we will live with it since the error is already out.